### PR TITLE
chore(journeys): add coverage for chrome on windows 10

### PR DESCRIPTION
**Pull request by moriahmaney**
PR also contains commits for stability changes that were necessary to get all tests passing, and optimizations to reduce total runtime of the jenkins pipeline.

### CODE REVIEW

**Comment by adamweeks** [comment]
Deploy preview for *hopeful-hamilton-4090b9* ready!

Built with commit f987ec8b2b67add397eee70cc6195bfad3b6eded

https://deploy-preview-656--hopeful-hamilton-4090b9.netlify.com

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "test:automation:space": "wdio wdio.conf.js --suite space",
     "test:automation:recents": "wdio wdio.conf.js --suite recents",
     "test:tap": "cross-env TAP=true wdio wdio.conf.js --suite tap",
+    "test:integration": "wdio wdio.conf.js --suite recents ; sleep 60 ; wdio wdio.conf.js --suite multiple ; sleep 60 ; wdio wdio.conf.js --suite oneOnOne ; sleep 60 ; wdio wdio.conf.js --suite space",
```
Comment (adamweeks) [inline package.json]: Will the `sleep` command work on windows? Anything in the package scripts should be cross-platform.

File: test/journeys/lib/test-helpers/index.js [hunk 2]
```diff
@@ -11,14 +11,14 @@ import uuid from 'uuid';
  * @returns {void}
  */
 // eslint-disable-next-line import/prefer-default-export
-export function moveMouse(aBrowser, selector, offsetX = 0, offsetY = 0) {
```
Comment (adamweeks) [inline test/journeys/lib/test-helpers/index.js]: Bye 😢

File: test/journeys/specs/space/global/basic.js [hunk 3]
```diff
@@ -121,16 +121,13 @@ describe('Widget Space', () => {
...
-      const spaceWidget = '.ciscospark-space-widget';
       browserLocal.execute((localAccessToken, spaceId) => {
         const options = {
           accessToken: localAccessToken,
           spaceId
         };
         window.openSpaceWidget(options);
       }, marty.token.access_token, conversation.id);
-
-      browserLocal.waitForVisible(spaceWidget);
```
Comment (adamweeks) [inline test/journeys/specs/space/global/basic.js]: We don't need to make sure the widget actually loads before continuing?

Comment (moriahmaney) [reply test/journeys/specs/space/global/basic.js]: @adamweeks the load is redundant as we have multiple other waits in each test. for some reason, having the waitForVisible in the before() block is what was causing the test time to explode in length

File: package.json (hunk 1 above)
Comment (moriahmaney) [reply package.json]: good catch, let me double-check this and get back to you...

File: test/journeys/specs/space/global/basic.js (hunk 3 above)
Comment (adamweeks) [reply test/journeys/specs/space/global/basic.js]: Cool, just making sure!

**Pull request merged**